docs/support.md

```diff
@@ -0,0 +1,119 @@
+# **Support Reference Directory**
+
+The purpose of this page is to provide a directory for support contacts. Everyone currently working with the Decred project has their Slack username listed.
+
+## Join Us on Slack
+
+Join us on Slack for support. You can receive an invite to the Decred Slack by visiting [https://slack.decred.org](https://slack.decred.org). Once you've signed up, you can join using your web browser by visiting [https://decred.slack.com](https://decred.slack.com) or by downloading one of Slack's desktop or mobile applications.
+
+## Development Support
+
+**@jy-p: Boss Man** <br />
+Project Leader <br />
+CEO of Company0
+
+**@davecgh: Lead Developer** <br />
+Consensus Expert<br />
+General Support
+
+**marcop (IRC only): Developer** <br />
```

Comment:
Since you mention someone who is irc only maybe you should mention the irc details on the top too.